### PR TITLE
fix: add Rydberg local schemas

### PR DESCRIPTION
**Pull request by math411**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/amazon-braket/amazon-braket-schemas-python/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/amazon-braket/amazon-braket-schemas-python/blob/main/CONTRIBUTING.md#commit-your-change)
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/amazon-braket/amazon-braket-schemas-python/blob/main/README.md) and [API docs](https://github.com/amazon-braket/amazon-braket-schemas-python/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/braket/ir/ahs/__init__.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 from braket.ir.ahs.atom_arrangement import AtomArrangement  # noqa: F401
 from braket.ir.ahs.driving_field import DrivingField  # noqa: F401
 from braket.ir.ahs.hamiltonian import Hamiltonian  # noqa: F401
-from braket.ir.ahs.shifting_field import LocalDetuning  # noqa: F401
 from braket.ir.ahs.physical_field import PhysicalField  # noqa: F401
 from braket.ir.ahs.program_v1 import Program, Setup  # noqa: F401
+from braket.ir.ahs.shifting_field import LocalDetuning  # noqa: F401
```
Comment (rchilaka-amzn) [inline src/braket/ir/ahs/__init__.py:19]: local_detuning, not shifting_field

File: src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py [hunk 2]
```diff
@@ -107,17 +107,50 @@ class RydbergGlobal(BaseModel):
     timeMax: Decimal
 
 
+class RydbergLocal(BaseModel):
+    """
+    Constraints for the parameters of the local shifting field
```
Comment (rchilaka-amzn) [inline src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py:112]: Not shifting field. Local detuning.

File: src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py [hunk 3]
```diff
@@ -300,14 +333,43 @@ class PerformanceRydbergGlobal(BaseModel):
     rabiAmplitudeRampCorrection: List[RabiCorrection]
 
 
+class PerformanceRydbergLocal(BaseModel):
+    """
+    Performance metrics for the local shifting field
```
Comment (rchilaka-amzn) [inline src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py:338]: Local detuning field

**Review by rchilaka-amzn — CHANGES_REQUESTED** [review]

File: src/braket/ir/ahs/__init__.py (hunk 1 above)
Comment (math411) [reply src/braket/ir/ahs/__init__.py:19]: Fixed

File: src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py (hunk 3 above)
Comment (math411) [reply src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py:338]: Fixed

File: src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py [hunk 4]
```diff
@@ -300,14 +333,43 @@ class PerformanceRydbergGlobal(BaseModel):
...
+    Attributes:
+        detuningErrorRms(Decimal):
```
[the remark this answers is not in the document]
Comment (rchilaka-amzn) [reply src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py:340]: nvm, these look fine. Sorry!

**Review by maolinml — APPROVED** [review]

**Review by rchilaka-amzn — APPROVED** [review]

**Review by speller26 — APPROVED** [review]

**Pull request merged**